cryptofeed/standards.py

```diff
@@ -30,10 +32,10 @@
 _exchange_to_std = {}
 
 
-def load_exchange_pair_mapping(exchange: str):
+def load_exchange_pair_mapping(exchange: str, key_id: str):
```

Comment:
why did this change?

Reply:
Idem as above
WHALE_ALERT feed required an API key even to query recognized coins.
So I need to forward the API key to all functions that call `gen_pairs()`.
This is the case for `load_exchange_pair_mapping`.